docs/visualization/lupton_rgb.rst

```diff
@@ -16,15 +20,51 @@ generate a color PNG file with the default (arcsinh) scaling:
 >>> imageR = np.random.random((100,100))
 >>> imageG = np.random.random((100,100))
 >>> imageB = np.random.random((100,100))
->>> image = lupton_rgb.make_lupton_rgb(image_r, image_g, image_b, filename='randoms.png')
+>>> image = make_lupton_rgb(image_r, image_g, image_b, filename='randoms.png')
 
 This method requires that the three images be aligned and have the same pixel
-scale and size.
+scale and size. Changing ``minimum`` will change the black level, while
+``stretch`` and ``Q`` will change how the values between black and white are
+scaled.
 
-Changing ``minimum`` and ``data_range`` will change the black (``minimum``) and white
-(``minimum+data_range``) levels of the resulting image, while changing ``Q`` will
-change how the values between black and white are scaled.
+For a more in-depth example, download the `g`_, `r`_, `i`_ SDSS frames of the
+area around the Hickson 88 group and try the example below (requires the
+`reproject package`_, installable via pip), and compare it with Figure 1 of
+Lupton et al. (2004):
 
-The SDSS SkyServer color images were made with this technique.
+.. _reproject package: https://reproject.readthedocs.io/
 
-.. _Lupton et al. (2004): http://adsabs.harvard.edu/abs/2004PASP..116..133L
+.. _g: http://dr13.sdss.org/sas/dr13/eboss/photoObj/frames/301/1737/5/frame-g-001737-5-0039.fits.bz2
+.. _r: http://dr13.sdss.org/sas/dr13/eboss/photoObj/frames/301/1737/5/frame-r-001737-5-0039.fits.bz2
+.. _i: http://dr13.sdss.org/sas/dr13/eboss/photoObj/frames/301/1737/5/frame-i-001737-5-0039.fits.bz2
+
+.. doctest-skip::
+
+   import numpy as np
+   from astropy.visualization import make_lupton_rgb
+   from astropy.io import fits
+   from reproject import reproject_interp
+
+   # Read in the three images downloaded from here:
+   # g: http://dr13.sdss.org/sas/dr13/eboss/photoObj/frames/301/1737/5/frame-g-001737-5-0039.fits.bz2
+   # r: http://dr13.sdss.org/sas/dr13/eboss/photoObj/frames/301/1737/5/frame-r-001737-5-0039.fits.bz2
+   # i: http://dr13.sdss.org/sas/dr13/eboss/photoObj/frames/301/1737/5/frame-i-001737-5-0039.fits.bz2
+   g = fits.open('frame-g-001737-5-0039.fits.bz2')[0]
+   r = fits.open('frame-r-001737-5-0039.fits.bz2')[0]
+   i = fits.open('frame-i-001737-5-0039.fits.bz2')[0]
+
+   # remap r and i onto g
+   r_new, r_mask = reproject_interp(r, g.header)
+   i_new, i_mask = reproject_interp(i, g.header)
+
+   # zero out the unmapped values
+   i_new[np.logical_not(i_mask)] = 0
+   r_new[np.logical_not(r_mask)] = 0
+
+   # red=i, green=r, blue=g
+   # make a file with the default scaling
+   rgb_default = make_lupton_rgb(i_new, r_new, g.data, filename="ngc6976-default.jpeg")
+   # this scaling is very similar to the one used in Lupton et al. (2004)
+   rgb = make_lupton_rgb(i_new, r_new, g.data, Q=10, stretch=0.5, filename="ngc6976.jpeg")
```

Comment:
Just replace ``.. doctest-skip`` by ``..plot`` and then use ``plt.imshow`` to show the result. You'll need to also add an ``import matplotlib.pyplot as plt``